src/client/MapleCharacter.java

```diff
@@ -1144,7 +1132,16 @@ public static MapleCharacter loadCharFromDB(int charid, MapleClient client, bool
                 ret.mount = new MapleMount(ret, mount != null ? mount.getItemId() : 0, 80001000, rs.getByte("Fatigue"), rs.getByte("Level"), rs.getInt("Exp"));
                 ps.close();
                 rs.close();
-
+				
+				ps = con.prepareStatement("SELECT * FROM favorites WHERE `accountid` = ?");
```

Comment:
Maybe somebody needs to specify formatting rules; this one is spaces and tabs combined